src/test/TimelockHook.t.sol

```diff
@@ -7,15 +7,15 @@ import {IAludelV3} from "../contracts/aludel/AludelV3.sol";
 import {TimelockHook} from "../contracts/aludel/TimelockHook.sol";
 import "forge-std/console2.sol";
 
-contract PowerSwitchTest is Test {
+contract TimelockHookTest is Test {
 
     function test_n_days(uint8 n) public {
         uint256 lockTime = uint256(n) * 1 days;
         IAludelV3.StakeData memory stake = IAludelV3.StakeData({amount: 100, timestamp: 1});
         TimelockHook hook = new TimelockHook(lockTime);
         vm.expectRevert(TimelockHook.TimelockNotElapsed.selector);
-        hook.unstakeAndClaimPost(stake);
+        hook.unstakeAndClaimPost(stake, address(0));
```

Comment:
it would be nice to know how this change affect gas measurements, what do you think about include the gas snapshots to this pr?

Reply:
as of this PR there isn't any integration test of the Aludel that calls the stake function using a hook, I added it but it doesn't provide huge insights.